### PR TITLE
Fix riskyUser resource and apis to match metadata. 

### PR DESCRIPTION
**Pull request by finsharp**
Fix missing response pairs for various entities.
Add missing entities {riskuseractivity and riskyuserhistoryitem} required by riskyuser entity.
Remove risk.md as its a duplicated of identityRiskEvent and doesn't exist in metadata.
Correct types and schema structures.

### CODE REVIEW

**Comment by jthake-msft** [comment]
Docs Build status updates of commit _[eda6b61](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4302/commits/eda6b611473737069d7805af46c767e641130661)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/driveitem-unfollow.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/privilegedrolesettings-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/privilegedrolesettings-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-confirmcompromised.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-confirmcompromised.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-dismiss.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-dismiss.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-list.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/risk.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/risk.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskuseractivity.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskuseractivity.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskyuser.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskyuser.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskyuserhistoryitem.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskyuserhistoryitem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201904021212102170-4302%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2BiRlgyleNJfhEcS5XogAiaOreLoU%2Ff%2B3R5EJXzLwbUQ%3D&st=2019-04-02T12%3A10%3A27Z&se=2019-05-03T12%3A15%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/driveitem-unfollow.md [hunk 1]
```diff
@@ -55,8 +55,13 @@ This example unfollows an item identified by `{item-id}`.
 ```http
 DELETE /me/drive/following/{item-id}
 ```
-
-
+<!-- {
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/driveitem-unfollow.md]: @finsharp , your addition here is appropriate.
On the other hand, I am curious if you have run API Doctor to these changes locally?
By adding this HTML comment block with blockType set to "response", the topic ends up having 1 blockType set to "request" and 2 blockType set to "response" (the other one is under the "Response" section above, in line 43).
FYI I am going to remove lines 43 to 47 in the "Response" section. This will solve this anticipated API Doctor problem. And, lines 43 to 47 do not follow the existing doc guidelines either - under the "Response" section, the original author should described the returned status code in plain English and not format it as an http block.

@finsharp , the takeaway from this is I recommend you to run API Doctor locally with each of these PRs you're making going into beta. This would be for the similar reason for my earlier request after I reviewed and fixed the content in the earlier bigger PR #3919 .

**Comment by jthake-msft** [comment]
Docs Build status updates of commit _[01103ff](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4302/commits/01103ff5e005ecb2417a6a3ed4f84e927224d0ac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/driveitem-unfollow.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/privilegedrolesettings-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/privilegedrolesettings-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-confirmcompromised.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-confirmcompromised.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-dismiss.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-dismiss.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-list.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/risk.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/risk.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskuseractivity.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskuseractivity.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskyuser.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskyuser.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskyuserhistoryitem.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskyuserhistoryitem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201904021828092100-4302%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=j%2FLtq%2BlkIhubVVKWfkFplmhYII%2F31Wag5ZP%2BxdyCIxU%3D&st=2019-04-02T18%3A26%3A24Z&se=2019-05-03T18%3A31%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/driveitem-unfollow.md (hunk 1 above)
Comment (finsharp) [reply api-reference/beta/api/driveitem-unfollow.md]: @angelgolfer-ms I ran ApiDoctor and that's how I caught the issue. I think my resolution is what wasn't appropriate. I should have removed it as you have noted and done. Thanks for the heads up.

File: api-reference/beta/api/riskyusers-confirmcompromised.md [hunk 2]
```diff
@@ -66,7 +65,7 @@ Here is an example of the response.
 <!-- {
   "blockType": "response",
   "truncated": true,
-  "@odata.type": "microsoft.graph.riskyUsers"
+  "@odata.type": "microsoft.graph.riskyUser"
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/riskyusers-confirmcompromised.md]: FYI I removed this line altogether as this API doesn't return anything in the response body and therefore this line is not applicable.

File: api-reference/beta/api/riskyusers-confirmcompromised.md [hunk 3]
```diff
@@ -80,6 +79,5 @@ HTTP/1.1 204 NoContent
   "keywords": "",
   "section": "documentation",
   "tocPath": "",
-  "suppressions": [
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/riskyusers-confirmcompromised.md]: AFAIK, all the beta reference topics have the beta disclaimer that uses the docs.microsoft.com "note" display. API Doctor doesn't recognize that display and would generate an error, and that's why we inserted the error suppression.

If you remove this suppression, doesn't API Doctor generate an error that would block merging?
I didn't add it back; it's worthwhile for us to straighten this out please.

File: api-reference/beta/api/riskyusers-get.md [hunk 4]
```diff
@@ -60,12 +60,14 @@ Here is an example of the response.
 } -->
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/riskyusers-get.md]: I changed line 59 to "@odata.type": "microsoft.graph.riskyUser", singular riskyUser, not plural riskyUsers.

File: api-reference/beta/api/riskyusers-list.md [hunk 5]
```diff
@@ -61,59 +61,64 @@ Here is an example of the response.
 } -->
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/riskyusers-list.md]: I changed the HTML comment block preceding the example response (starting in line 58) to specify the response should be a collection of riskyUser type:
  "isCollection": true,
  "@odata.type": "microsoft.graph.riskyUser"

I am not sure if the original line that specifies the riskyUsers entitySet would work with API Doctor.
@finsharp , did you run API Doctor on this topic and did the original line with the riskyUsers entitySet work? 
I doubt it as we only define the entityType riskyUser and not the entitySet riskyUsers.

File: api-reference/beta/resources/riskuseractivity.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+|:---------------|:--------|:----------|
+| eventTypes | riskEventType collection | |
+| detail     | riskDetail  | none, adminGeneratedTemporaryPassword, userPerformedSecuredPasswordChange, userPerformedSecuredPasswordReset, adminConfirmedSigninSafe, aiConfirmedSigninSafe, userPassedMFADrivenByRiskBasedPolicy, adminDismissedAllRiskForUser, adminConfirmedSigninCompromised, hidden, adminConfirmedUserCompromised, unknownFutureValue  |       |
+
+## JSON representation
+
+<!-- {
+  "blockType": "resource",
+  "optionalProperties": [ ],
+  "@type": "microsoft.graph.riskUserActivity"
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/riskuseractivity.md]: This should be "@odata.type", not "@type".

File: api-reference/beta/resources/riskyuser.md [hunk 7]
```diff
@@ -19,10 +19,10 @@ For more information about risk events, see [Azure Active Directory Identity Pro
 
 | Method   | Return Type|Description|
 |:---------------|:--------|:----------|
-|[List riskyUsers](../api/riskyusers-list.md) | [riskyUsers](riskyUser.md) |List risky users and their properties.|
-|[Get riskyUsers](../api/riskyusers-get.md) | [riskyUsers](riskyUser.md)|Get a specific risky user and its properties.|
-|[Confirm riskyUsers compromised](../api/riskyusers-confirmcompromised.md)|Confirm a risky user as compromised.|
-|[Dismiss riskyUsers](../api/riskyusers-dismiss.md)|Dismiss the risk of a risky user.|
+|[List riskyUsers](../api/riskyusers-list.md) | [riskyUser](riskyUser.md) collection|List risky users and their properties.|
+|[Get riskyUsers](../api/riskyusers-get.md) | [riskyUser](riskyUser.md) collection|Get a specific risky user and its properties.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/riskyuser.md]: The second method should be "get riskyUser" singular.

File: api-reference/beta/resources/riskyuser.md [hunk 8]
```diff
@@ -19,10 +19,10 @@ For more information about risk events, see [Azure Active Directory Identity Pro
...
+|[Confirm riskyUsers compromised](../api/riskyusers-confirmcompromised.md)| |Confirm a risky user as compromised.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/riskyuser.md]: Add "None" to the "Return type" column if the method doesn't return any object in the response payload.

**Comment by jthake-msft** [comment]
Docs Build status updates of commit _[ba41542](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4302/commits/ba415426327cb24f085729f4eaec0a7a2670ae0b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/driveitem-unfollow.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/privilegedrolesettings-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/privilegedrolesettings-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-confirmcompromised.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-confirmcompromised.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-dismiss.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-dismiss.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/riskyusers-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/api/riskyusers-list.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/risk.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/risk.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskuseractivity.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskuseractivity.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskyuser.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskyuser.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/riskyuserhistoryitem.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/riskyusers/api-reference/beta/resources/riskyuserhistoryitem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201904030005439642-4302%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rDgPQvNv%2BNQLAl7iod6JUC%2BnRHnZc9usAiEIIxvb8h0%3D&st=2019-04-03T00%3A04%3A09Z&se=2019-05-04T00%3A09%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/riskyusers-confirmcompromised.md (hunk 3 above)
Comment (finsharp) [reply api-reference/beta/api/riskyusers-confirmcompromised.md]: Hi Angela, the suppression will no longer be necessary due to this [PR](https://github.com/OneDrive/apidoctor/pull/14). We have a tool capable of stripping them out. I stripped it out to see the final results of my tests.

File: api-reference/beta/api/riskyusers-get.md (hunk 4 above)
Comment (finsharp) [reply api-reference/beta/api/riskyusers-get.md]: Noted.

**Comment by angelgolfer-ms** [comment]
@finsharp , can you tell me your strategy? This PR 4302 has the very similar content updates as those for the same files in PR 3919, which I also reviewed yesterday. 
Are you going to merge PR 4302 into master any time now?
If so, would you consider merging PR 4302 first into master, and then do an "update branch" for PR 3919, so that you and I don't manually repeat / review the same content changes for PR 4302 and PR 3919?

That being said, there's at least one area right now that the same file in PR 3919 has more complete content - in riskuseractivity.md, the "Properties" section, I filled out the Description column for the eventTypes property. In this PR 4302, the eventTypes property has a blank description.  That's why I want to understand the purpose of having the same files manually updated in PR 4302 and PR 3919.

**Comment by finsharp** [comment]
> @finsharp , can you tell me your strategy? This PR 4302 has the very similar content updates as those for the same files in PR 3919, which I also reviewed yesterday.
> Are you going to merge PR 4302 into master any time now?
> If so, would you consider merging PR 4302 first into master, and then do an "update branch" for PR 3919, so that you and I don't manually repeat / review the same content changes for PR 4302 and PR 3919?
> 
> That being said, there's at least one area right now that the same file in PR 3919 has more complete content - in riskuseractivity.md, the "Properties" section, I filled out the Description column for the eventTypes property. In this PR 4302, the eventTypes property has a blank description. That's why I want to understand the purpose of having the same files manually updated in PR 4302 and PR 3919.

**Comment by finsharp** [comment]
@angelgolfer-ms PR3919 was growing a bit hairy for minor fixes. the workflow I have been using is, make any minor fixes that are focused on one area of docs in a minor PR, have that reviewed and merged to master. Merge master to PR3919.

**Comment by angelgolfer-ms** [comment]
> PR3919 was growing a bit hairy for minor fixes. the workflow I have been using is, make any minor fixes that are focused on one area of docs in a minor PR, have that reviewed and merged to master. Merge master to PR3919.

@finsharp , I think this is a sound approach, thanks for confirming.
Then we have to deal with the situation on hand:
If you look at PR 3919, search for the following [comment ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3919#discussion_r272080468) that I added Wednesday overnight (our time):
"I filled in the possible values for eventTypes in the Description column.
I also updated the riskEventType values in enums.md. It was missing 5 possible values."

There are updates in the riskyUser set of API files for PR 3919, that are not in PR 4302:
- In PR 3919, in riskuseractivity.md, the "Properties" section, the Description column for the eventTypes property is populated with the possible values. 
- In this PR 4302, the eventTypes property has a blank description. 

So after we merge PR 4302 to master, and merge from master to PR 3919, we want to make sure that we don't lose what's already in PR 3919, like this example of the Description column for the eventTypes property.

I think Github would call out there's a merge conflict when we next refresh PR #3919 . I just want us to be aware and resolve the merge conflict appropriately.
